### PR TITLE
Use 'quill sns' instead of 'sns-quill'.

### PR DESCRIPTION
**Pull request by dskloetd**
# Motivation

I'm trying to make snsdemo work on my M1 Mac for local testing.
sns-quill is giving me
```
Library not loaded: /usr/local/opt/openssl@1.1/lib/libssl.1.1.dylib
```
For other tools that gave this error, I replaced them with version that have openssl statically linked, but I couldn't find one for sns-quill and supposedly sns-quill functionality is now provided by `quill sns`.
However, snsdemo was still using the old swap flow with a single command, which is not supported by `quill sns`.

Also, `quill sns pay` takes an amount in e8s while `sns-quill` takes an amount in whole ICP.

# Changes

1. Remove installation of `sns-quill` from `dfx-sns-demo-install`.
2. Delete `bin/dfx-software-sns-quill-install`.
3. Use `quill sns` for creating the message and `quill send` for sending the message instead of using `sns-quill`.
4. Move sending the message from `dfx-sns-sale-buy` to `dfx-sns-quill`.
5. Make separate `quill sns new-sale-ticket` and `quill sns pay` calls.
6. Convert the amount passed to dfx-sns-sale-buy to e8s, or if `max` is passed, no longer convert from e8s.

### CODE REVIEW

**Review by lmuntaner — DISMISSED** [review]

**Comment by dskloetd** [comment]
This doesn't work yet. `sns-quill` has a (old?) swap command but it seems like `quill sns` requires going through the ticket flow. I'll look into it more.

**Comment by dskloetd** [comment]
This is now a bigger change, but it also includes the new ticket swap flow.

**Review by bitdivine — APPROVED** [review]
Looks plausible.  Thank you!

File: bin/dfx-sns-quill [hunk 1]
```diff
@@ -27,7 +27,24 @@ dfx-sns-quill-canister-ids --network "$DFX_NETWORK" >"$SNS_QUILL_CANISTER_IDS_FI
 PEM_PATH="$(dfx-identity-pem --identity "$DFX_IDENTITY")"
 
 # Call sns-quill
-sns-quill --canister-ids-file "$SNS_QUILL_CANISTER_IDS_FILE" --pem-file "$PEM_PATH" "${@}"
+set quill sns --canister-ids-file "$SNS_QUILL_CANISTER_IDS_FILE" --pem-file "$PEM_PATH" "${@}"
+echo "${@}"
+MESSAGE="$(${@})"
```
[the remark this answers is not in the document]
Comment (dskloetd) [reply bin/dfx-sns-quill]: Thanks

**Pull request merged**